test/test.js

```diff
@@ -5,15 +5,16 @@ const ASSERT = require('assert'),
   SPEC_GEN = require('../server');
 
 const PORT = 3000,
-  BASE_URL = `http://localhost:3000/?${process.env.AUTHORIZATION}`,
+  AUTHORIZATION = encodeURIComponent(process.env.AUTHORIZATION),
+  BASE_URL = 'http://localhost:3000/',
   NO_URL = '&type=foo&URL=notice-that-its-in-uppercase',
```

Comment:
Because you changed `BASE_URL` (now it does *not* include `?`), you have to change also the first char of all strings in lines 10&ndash;17 so that they all start with `?` instead of `&`.